Directory.Build.props

```diff
@@ -84,8 +86,9 @@
     -->
     <Otherwise>
       <PropertyGroup>
-        <DefaultNetCoreTargetFramework>net8.0</DefaultNetCoreTargetFramework>
-        <DefaultNetCoreTargetFrameworks>$(DefaultNetCoreTargetFramework)</DefaultNetCoreTargetFrameworks>
+        <NetVS>net8.0</NetVS>
+        <NetVSCode>net9.0</NetVSCode>
+        <NetVSAndVSCode>$(NetVS);$(NetVSCode)</NetVSAndVSCode>
```

Comment:
Consider copying the approach taken in https://github.com/dotnet/roslyn/blob/b6ed7a5f13a3f00e7edf8cc4fab0d4524251f056/eng/targets/TargetFrameworks.props#L14-L22 even more. I.e., instead of setting these in the `<Otherwise>` block, declare and set them in the main `<PropertyGroup>` above and like the `<Choose>` just override them. That way, they're easy to update.

Also, consider renaming `DefaultNetFxTargetFramework`, since you're unifying the naming convention.

Reply:
> Consider copying the approach taken in [dotnet/roslyn@b6ed7a5/eng/targets/TargetFrameworks.props#L14-L22](https://github.com/dotnet/roslyn/blob/b6ed7a5f13a3f00e7edf8cc4fab0d4524251f056/eng/targets/TargetFrameworks.props#L14-L22) even more. I.e., instead of setting these in the <Otherwise> block, declare and set them in the main <PropertyGroup> above and like the <Choose> just override them. That way, they're easy to update.

SGTM :) 

> Also, consider renaming DefaultNetFxTargetFramework, since you're unifying the naming convention.

To `NetFxVS`?

Reply:
> To `NetFxVS`?

Perfect!